releases/release-1.19/README.md

```diff
@@ -0,0 +1,94 @@
+# Kubernetes 1.19
+
+
+#### Links
+
+* [This document](https://git.k8s.io/sig-release/releases/release-1.19/README.md)
+* [Release Team](https://git.k8s.io/sig-release/releases/release-1.19/release_team.md)
+* [Meeting Minutes](http://bit.ly/k8s119-releasemtg) (join [kubernetes-sig-release@] to receive meeting invites)
+* [v1.19 Release Calendar][k8s119-calendar]
+* Contact: [#sig-release] on slack, [kubernetes-release-team@] on e-mail
+* [Internal Contact Info]() (accessible only to members of [kubernetes-release-team@])
+
+#### Tracking docs
+
+* [Enhancements Tracking Sheet](http://bit.ly/k8s-1-19-enhancements)
+* Bug Triage Tracking Sheet: TODO
+* CI Signal Report: TODO
+* [Retrospective Document][Retrospective Document]
+* [kubernetes/sig-release v1.19 milestone](https://github.com/kubernetes/kubernetes/milestone/46)
+
+#### Guides
+
+* [Targeting Issues and PRs to This Milestone](https://git.k8s.io/community/contributors/devel/sig-release/release.md)
+* [Triaging and Escalating Test Failures](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md#troubleshooting-a-failure)
+
+## TL;DR
+
+The 1.19 release cycle is proposed as follows:
+
+- **Monday, April 13**: Week 1 - Release cycle begins
+- **Tuesday, May 5**: Week 4 - [Enhancements Freeze]
+- **Thursday, June 11**: Week 9 - [Code Freeze]
+- **Tuesday, June 23**: Week 11 - Docs must be completed and reviewed
+- **Tuesday, June 30**: Week 12 - Kubernetes v1.19.0 released
+
+
+## Timeline
+
+
+| **What** | **Who** | **Apr** | **May** | **Jun** | **Jul** | **WEEK** | **CI SIGNAL** |
```

Comment:
@onlydole -- Here's a rough draft of what I'm thinking about on the Branch Management side:


| **What** | **Who** | **Apr** | **May** | **Jun** | **Jul** | **WEEK** | **CI SIGNAL** |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Start of Release Cycle | Lead | Mon 13 | | | | week 1 | [master-blocking] |
| 1.19.0-alpha.2 released | Branch Manager | Tue 21 | | | | week 2 | |
| **Begin [Enhancements Freeze]** (EOD PST) | Enhancements Lead | | Tues 5 | | | week 4 | [master-blocking], [master-informing] |
| 1.19.0-alpha.3 released | Branch Manager | | Tue 5 | | | week 4 | |
| 1.19.0-beta.0 released | Branch Manager | | Tue 19 | | | week 6 | |
| 1.19.0-beta.1 released | Branch Manager | | Tue 26 | | | week 7 | |
| 1.19.0-beta.2 released | Branch Manager | | | Tue 2 | | week 8 | |
| **Begin [Code Freeze]** (EOD PST) | Branch Manager | | | Thu 11 | | week 9 | |
| 1.19.0-rc.1 released | Branch Manager | | | Thu 11 | | week 9 | |
| release-1.19 branch created | Branch Manager | | | Thu 11 | | week 9 | |
| release-1.19 jobs created | Branch Manager | | | Thu 11 | | week 9 | |
| release-1.15 jobs removed | Branch Manager | | | Thu 11 | | week 9 | |
| 1.19.0-rc.2 released | Branch Manager | | | Tue 16 | | week 10 | |
| **Cherry Pick Deadline** (EOD PST) | Branch Manager | | | Tue 23 | | week 11 | |
| 1.19.0-rc.3 released | Branch Manager | | | Thu 25 | | week 11 | |
| **v1.19.0 released** | Branch Manager | | | Tue 30 | | week 12 | |

Consider this alpha as well. Just putting it up so we can start discussing removing fast-forwards (ref: https://github.com/kubernetes/release/issues/1207).

tl;dr:
- cut alphas every two weeks, then:
- cut 3 betas weekly on Tuesdays, then:
- start Code Freeze, which would now be signified by cutting RC1 and the release-1.19 branch
- cherry pick deadline happens on a week after RC2 and BEFORE RC3
- RC3 is released
- 1.19.0 is released (and likely has the same content as RC3)

By doing the branch cut at Code Freeze, that means:
- `master` immediately opens for Kubernetes 1.20 and Code Thaw is obsolete
- any content that needs to land in 1.19 needs to be cherry picked

@kubernetes/release-engineering -- Thoughts on this?

Reply:
Feels sensible to me.

Reply:
I think the Cherry Pick deadline could still be Thu 25.

Reply:
@justaugustus and @tpepper - I have updated the schedule to reflect these changes, let me know if you like how that looks now.

Reply:
Sounds good to me! :) :+1:

Reply:
sgtm as well, thanks!

Reply:
LGTM from my side too. Thank you! :)

Reply:
looks good! lgtm 👍